src/oc/web/actions/user.cljs

```diff
@@ -42,8 +43,30 @@
 (defn jwt-refresh []
   (api/jwt-refresh update-jwt logout))
 
-;; Login
+;; API Entry point
+(defun entry-point-get-finished
```

Comment:
@thepug You don't need `defun` for this signature, you can use `defn` with this method definitions.